etc/profile-m-z/nicotine.profile

```diff
@@ -47,7 +52,7 @@ seccomp
 tracelog
 
 disable-mnt
-private-bin nicotine,python2*
+#private-bin bash,nicotine,python2*,python3*,sh
```

Comment:
```suggestion
#private-bin bash,nicotine,python2*,sh
```

Reply:
@rusty-snake During the help attempt on IRC I at one time suggested to try adding python3* to private-bin, besides including its allow file. Later on the user in question confirmed that private-bin kept breaking nicotine so I decided to comment it. Thanks for the suggestion, I'll take it out again.